src/Security/Permission.php

```diff
@@ -706,6 +709,7 @@ public static function get_label_for_permission($perm)
      */
     protected static function traverse_declared_permissions($declared, &$list)
     {
+        Deprecation::notice('4.4.0');
```

Comment:
Add a standardised message here and to the phpdoc

Reply:
Updated